### PR TITLE
Add a notice for using regional init action buckets.

### PR DESCRIPTION
**Pull request by medb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,33 @@ When creating a [Google Cloud Dataproc](https://cloud.google.com/dataproc/) clus
...
+```
+
+During development, you can create Dataproc cluster using Dataproc-provided regional initialization actions buckets:
+
+```bash
+REGION=<region>
+CLUSTER=<cluster_name>
+gcloud dataproc clusters create ${CLUSTER} \
+    --region ${REGION} \
+    --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/presto/presto.sh
```
Comment (DanSedov) [inline README.md]: I've notived the word region confuses some people and they keep trying "us-east1-a" for instance. Consider adding an example of a region name or gcloud command to list regions

File: beam/README.md [hunk 2]
```diff
@@ -123,9 +128,9 @@ METADATA+=",flink-start-yarn-session=true"
 METADATA+=",flink-snapshot-url=${FLINK_SNAPSHOT}"
 
 gcloud dataproc clusters create "${CLUSTER_NAME}" \
-  --initialization-actions="${INIT_ACTIONS}" \
-  --image-version="1.2" \
-  --metadata="${METADATA}"
+    --initialization-actions "${INIT_ACTIONS}" \
+    --image-version "1.2" \
```
Comment (DanSedov) [inline beam/README.md]: Why 1.2?

File: bigdl/README.md [hunk 3]
```diff
@@ -10,11 +10,16 @@ More information [project's website](https://analytics-zoo.github.io)
...
 
 ```
-gcloud dataproc clusters create <CLUSTER_NAME> \
-    --initialization-actions gs://$MY_BUCKET/bigdl/bigdl.sh \
+REGION=<region>
+CLUSTER_NAME=<cluster_name>
+gcloud dataproc clusters create ${CLUSTER_NAME} \
+    --region ${REGION} \
+    --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/bigdl/bigdl.sh \
     --initialization-action-timeout 10m
```
Comment (DanSedov) [inline bigdl/README.md]: Should we call out why the 10m timeout? It seems to be more than just an example of specifying a timeout

File: connectors/README.md [hunk 4]
```diff
@@ -42,15 +47,21 @@ For example:
...
+  REGION=<region>
+  CLUSTER_NAME=<cluster_name>
+  gcloud dataproc clusters create ${CLUSTER_NAME} \
+      --region ${REGION} \
+      --initialization-actions gs://goog-dataproc-initialization-actions-${REGION}/connectors/connectors.sh \
       --metadata gcs-connector-version=1.7.0
   ```
 * if Google Cloud Storage connector 1.8.0 version is specified and BigQuery connector version is not
   specified, then only Google Cloud Storage connector will be updated to 1.8.0 version and BigQuery
   connector will be left intact:
```
Comment (DanSedov) [inline connectors/README.md]: (not sure if this is clearer) intact -> unchanged

File: README.md [hunk 5]
```diff
@@ -4,19 +4,33 @@ When creating a [Google Cloud Dataproc](https://cloud.google.com/dataproc/) clus
 
 ## How initialization actions are used
 
-Initialization actions are stored in a [Google Cloud Storage](https://cloud.google.com/storage) bucket and can be passed as a parameter to the `gcloud` command or the `clusters.create` API when creating a Cloud Dataproc cluster. For example, to specify an initialization action when creating a cluster with the `gcloud` command, you can run:
+Initialization actions should be stored in a [Google Cloud Storage](https://cloud.google.com/storage) bucket and can be passed as a parameter to the `gcloud` command or the `clusters.create` API when creating a Cloud Dataproc cluster. For example, to specify an initialization action when creating a cluster with the `gcloud` command, you can run:
```
Comment (aniket486) [inline README.md]: s/should/must.

File: beam/README.md (hunk 2 above)
Comment (medb) [reply beam/README.md]: I don't think that we got Beam to work on newer Dataproc images.

File: README.md (hunk 5 above)
Comment (medb) [reply README.md]: Done.

File: connectors/README.md (hunk 4 above)
Comment (medb) [reply connectors/README.md]: Done.

File: bigdl/README.md (hunk 3 above)
Comment (medb) [reply bigdl/README.md]: Added explanation at the top of the page.

File: README.md (hunk 1 above)
Comment (medb) [reply README.md]: Added full bucket name example and more information about Dataproc regional endopints to top-level README.md

**Pull request merged**